app/main.py

```diff
@@ -2,4 +2,66 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        if coords is None:
+            coords = [0, 0]
+        self.coords = coords
+
+    def go_forward(self, step: int = 1) -> list:
+        self.coords[1] += step
+        return self.coords
+
+    def go_back(self, step: int = 1) -> list:
+        self.coords[1] -= step
+        return self.coords
+
+    def go_right(self, step: int = 1) -> list:
+        self.coords[0] += step
+        return self.coords
+
+    def go_left(self, step: int = 1) -> list:
+        self.coords[0] -= step
+        return self.coords
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        if not coords:
+            coords = [0, 0, 0]
+        super().__init__(name, weight, coords)
```

Comment:
Do not use a multi-line if condition, perform assignment in one line here. You could use `or` here
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```